fastai2/vision/utils.py

```diff
@@ -44,4 +44,35 @@ def verify_image(fn):
 # Cell
 def verify_images(fns):
     "Find images in `fns` that can't be opened"
-    return L(fns[i] for i,o in enumerate(parallel(verify_image, fns)) if not o)
\ No newline at end of file
+    return L(fns[i] for i,o in enumerate(parallel(verify_image, fns)) if not o)
+
+# Cell
+def resize_image(file, max_size:None, dest=None, n_channels=3, ext=None,
```

Comment:
This should be =None. Also dest should be a require argument since this function does nothing if dest is None.

Reply:
why you don't use typing anymore?

Reply:
We don't, no, unless for patching/type-dispatching.